### PR TITLE
Widgetfactory mulibuffer point

### PR DESCRIPTION
**Pull request by hennie-k**

### CODE REVIEW

**Comment by hennie-k** [comment]
If not used:
image

If used:
image

File: geest/gui/widgets/geest_config_widget.py [hunk 1]
```diff
@@ -32,18 +31,14 @@ def create_widgets(self):
         self.setLayout(layout)
 
         print("Calling GeestWidgetFactory.create_widgets")
```
Comment (timlinux) [inline geest/gui/widgets/geest_config_widget.py:33]: No print

File: geest/gui/widgets/geest_config_widget.py [hunk 2]
```diff
@@ -68,58 +63,105 @@ def recursive_find_and_store_widgets(self, widget, depth=0):
...
-            ):
-                if use_key not in self.widgets:
-                    self.widgets[use_key] = {}
+            elif isinstance(widget, (QLineEdit, QSpinBox, QDoubleSpinBox, QComboBox, QgsMapLayerComboBox)):
                 self.widgets[use_key]["widget"] = widget
-                print(
-                    "  " * depth + f"Stored {type(widget).__name__} for key: {use_key}"
-                )
-
+                print("  " * depth + f"Stored {type(widget).__name__} for key: {use_key}")
```
Comment (timlinux) [inline geest/gui/widgets/geest_config_widget.py:73]: No print through out

**Comment by osundwajeff** [comment]
Hi @hennie-k can you also check on the black formater

**Comment by osundwajeff** [comment]
Hi @hennie-k can you also check on the black formater

**Comment by hennie-k** [comment]
I am closing this PR due to it being superseded by #301, which includes there and additional changes

**Pull request closed without merging**